oil_predictor_tests.py

```diff
@@ -87,4 +87,4 @@ def validateDateString(date_string):
 	predictions.append(yhat)
 	obs = test[t]
 	history.append(obs)
-	print('%s, Predicted = %f, Expected = %f, Difference = %f' % (str(data['Date'][t])[0:10], yhat, obs, (yhat-obs)) )
\ No newline at end of file
+	print('%s, Predicted = %f, Expected = %f, Difference = %f' % (str(data['Date'][t + size])[0:10], yhat, obs, (yhat-obs)) )
```

Comment:
Since this is python3 you could use fstrings here

Reply:
This was essentially pulled from the tutorial, I only slightly edited it, I agree that this is a messy.